packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -92,38 +57,22 @@ export class VaAccordionItem {
   }
 
   render() {
-    // IE11 is unable to run onSlotChange
-    // Check needed to populate Accordion Item Header information
-    // When using slot="headline" to set the information
-    const ieSlotCheckTag = this.el
-      .querySelector('[slot="headline"]')
-      ?.tagName.toLowerCase();
-    const ieSlotCheckHeader =
-      this.el.querySelector('[slot="headline"]')?.innerHTML;
-
-    const Header = () =>
-      h(
-        // TODO: Remove level & header prop & let the slot do its work
-        /* eslint-disable-next-line i18next/no-literal-string */
-        this.slotTag || ieSlotCheckTag || `h${this.level}`,
-        null,
+    const headerText = this.el.querySelector('[slot="headline"]').innerHTML.replace(/(<([^>]+)>)/ig, '');
+    return (
```

Comment:
What is this regex doing?

Reply:
That regex is removing the HTML tags from the header so that the header text can be used in the `aria-label` below